start-chromium-for-playr.sh

```diff
@@ -16,14 +16,18 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
```

Comment:
Laten we het script start-chrome-for-playr.sh omnoemen naat start-browser-for-playr.sh en in het script de keuze voor de browser laten maken op basis van wat geinstalleerd is en onze voorkeur voor de browser. Dan kan dit script verwijderd worden